### ISSUE TITLE
Toggle controls are not accessible

### ISSUE THREAD

**Issue by steverep** [ahmedkaludi/accelerated-mobile-pages#1691]
*I just installed your plugin today, and I am forced to defer using it or deactivate it altogether because of inaccessibility when using assistive technology, such as a screen reader.  I'll try to create separate issues for the biggest offenses as I see them.*

Each toggle on/off control is inaccessible.  I did not have time to did into your code, but the usual cause of this is relying on the clickability of `<label>` elements but hiding the actual checkbox or radios with CSS using `display: none;`.  This is a big mistake because it effectively hides it from assistive technology and makes it keyboard inaccessible in general.  Instead, use the CSS `position` property.  Here's an overly simplified example:

    // Make the container relative so its pieces are anchored
    .toggle {position: relative; }
    
    // Make the label  relative as well
    .toggle input[type="checkbox"] + label {position: relative;}
    
    // Make the input absolute so the adjacent label overlaps it
    .toggle input[type="checkbox"] {position: absolute;}

Depending on whether the label is left or right of the actual input field, you may need to use the `left` property as well, but you avoid that trouble by putting it first.

If this isn't applicable, let me know and I can dive into your code when I have time.

**Comment by steverep** [comment]
I found the offending code coming from the redux framework you are using:

    .redux-container-switch .switch-options input {
    	display: none;
    }

After inspection, it is confusing because the `.ios7-switch` class is already correctly hiding the input with absolute positioning.  The problem is that the redux rules are more specific and dominating those rules.  Is the actual switch being inserted by this plugin or is it coming from redux?

If it's coming from you, then just add a rule in your custom CSS for redux to override it, such as:

    .redux-container-switch .switch-options input {
    	display: initial ! important;
    }

If it's coming from redux, then you can still add that rule to correct it for your users, but they should correct the bug at the source.

**Comment by Sejiro-Hiko** [comment]
need to improve the toggle's CSS effect.
